descwl_shear_sims/galaxies.py

```diff
@@ -582,24 +583,49 @@ class WLDeblendGalaxyCatalog(object):
     buff: int, optional
         Buffer region with no objects, on all sides of image.  Ingored
         for layout 'grid'.  Default 0.
+    layout: str, optional
+
     """
-    def __init__(self, *, rng, coadd_dim, buff=0):
+    def __init__(self, *, rng, coadd_dim, buff=0, layout='random'):
```

Comment:
Here @esheldon. If you are trying to do a new `random_ring` simulation, you need to remember to pass this keyword here in addition to the other places `layout` is used.